src/main/scala/cognite/spark/v1/CdpConnector.scala

```diff
@@ -113,7 +113,7 @@ object CdpConnector {
       retryingSttpBackend(
         config.maxRetries,
         config.maxRetryDelaySeconds,
-        config.partitions,
+        config.parallelismPerPartition,
```

Comment:
third parameters is `maxParallelRequests` so should not be `partitions` here.
 ```scala
def retryingSttpBackend(
      maxRetries: Int,
      maxRetryDelaySeconds: Int,
      maxParallelRequests: Int = Constants.DefaultParallelismPerPartition,
      metricsPrefix: Option[String] = None
  )
```

Reply:
Nice catch.